Sources/Utility/SequenceTypeExtensions.swift

```diff
@@ -0,0 +1,28 @@
+/*
+This source file is part of the Swift.org open source project
+
+Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See http://swift.org/LICENSE.txt for license information
+See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+extension SequenceType {
+    @warn_unused_result
+    public func partition(@noescape include:(Generator.Element)->Bool) -> ([Generator.Element], [Generator.Element]) {
```

Comment:
can probably get rid of the same extension in Array then?
also `include: (Generator.Element) -> Bool)` would look better